recipes/conch_sounds/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "conch_sounds" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/conch-sounds-{{ version }}.tar.gz
+  sha256: 8dffbbf154baec8f265eaa0205b26f6edfcf0a39f94bb588cc540c8aa244fe30
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - librosa >=0.8.0
+    - pip
+    - pyraat
```

Comment:
Are you sure that all these extra dependencies are required for building the package?

Reply:
Librosa and pyraat are direct dependencies:
- https://github.com/mmcauliffe/Conch-sounds/search?q=librosa
- https://github.com/mmcauliffe/Conch-sounds/search?q=pyraat
I believe pip is required for building the package? I'm new to conda-forge so I could be mistaken.

Reply:
Yes but from what I understand they are only required when running the package, not when building it?

Yes, pip and python are almost always required.

Reply:
It turns out that removing librosa and pyraat caused build errors, I've kept them in the subsequent commits. I was able to remove similar dependencies from the polyglotdb meta.yaml